extensions/amp-a4a/0.1/test/test-real-time-config-manager.js

```diff
@@ -721,6 +721,15 @@ describes.realWin('real-time-config-manager', {amp: true}, env => {
       expect(rtc.rtcConfig_).to.deep.equal(expectedRtcConfig);
     });
 
+    it('should always clear RTC for a new consent state', () => {
```

Comment:
I think it would be better if we explicitly fail on a new enum values.

Reply:
done